pkgs/applications/misc/cherrytree/default.nix

```diff
@@ -1,8 +1,7 @@
-{ stdenv, fetchurl, pythonPackages, gettext }:
+{ stdenv, fetchurl, python, gettext }:
```

Comment:
Can be `pythonPackages` again.

Reply:
Addressed in 4bf36fa8a8ffd6cecfebc5a08ef63f89da69442e